### ISSUE TITLE
69-mixxx-usb-uaccess.rules causes long boot time Ubuntu 24.04

### ISSUE THREAD

**Issue by mrbrico** [mixxxdj/mixxx#13290]
### Bug Description

After upgrading to Ubuntu 24.04 boot times increased to around 4 minutes, I tracked this down to the above, commenting out KERNEL=="hiddev*", NAME="usb/%k", GROUP="uaccess" fixed the extremely long boot times.

By default this is enabled in the udev rule /usr/lib/udev/rules.d/69-mixxx-usb-uaccess.rules

# Only some distribuions require the below
KERNEL=="hiddev*", NAME="usb/%k", GROUP="uaccess"

Maybe default this should be commented out and enabled if needed...?

### Version

2.4.1

### OS

Ubuntu 24.04

**Comment by acolombier** [comment]
Hi @mrbrico , I haven't yet moved to 24.04 (waiting for the July SP). 
Would you be able to provide some system logs to try and narrow down the problem?

**Comment by mrbrico** [comment]
[log_messages_udev_disabled.log](https://github.com/user-attachments/files/15777875/log_messages_udev_disabled.log)
[log_messages_udev_enabled.log](https://github.com/user-attachments/files/15777876/log_messages_udev_enabled.log)

**Comment by acolombier** [comment]
Thanks for the logs and the kind words.

I'm not sure there is enough info to narrow down this bug. Could you try the following:
- Enable the udev rule
- Reload udev (`sudo udevadm control --reload-rules && sudo udevadm trigger`)
- Reboot (make sure the problem is still there
- Run `sudo journalctl --boot=-1` and share that. This will include both `dmesg`  and `systemd` logs (which should be used  to run udev daemon)

In the meantime, I have just checked on my end (PopOS 22.04) with `sudo systemctl status udev.service` and spotted that in the logs:
```
hiddev4: /usr/lib/udev/rules.d/69-mixxx-usb-uaccess.rules:61 Only network interfaces can be renamed, ignoring NAME="usb/%k".
```
I will look if I can fix that!

**Comment by mrbrico** [comment]
Journalctl boot log with the rule enabled...
[journalctl_boot.log](https://github.com/user-attachments/files/15778044/journalctl_boot.log)

**Comment by acolombier** [comment]
Yeah, this rule seems to be quite out of date, but that warning should block the system.

I can see you've started the reboot at `20:04:42` and the system rebooted at `20:04:46`, does that mean you were unable to reproduce the issue this time? 
Also, do you use a HID controller with Mixxx?

**Comment by mrbrico** [comment]
The delay is still there when the rule is enabled, its about 1 min delay now though... if I disable boot quiet in grub I can see systemd just stop for about 1 min before continuing. I could not find anything obvious in the logs and just went through errors I saw that may have affected boot time, plymouth / app amour /network manager until I tried the udev error above..
I use A&H k2 controllers or NI x1 controllers, also testing with serato dvs.. I never had the pioneer xdj 1000 mk2's working

**Comment by acolombier** [comment]
Thanks for the details. 
I will try to reproduce this in a VM soon and report my findings.

**Comment by alex-c3** [comment]
Can confirm this happens in Ubuntu 24.04, Mixxx and Ubuntu up-to-date with apt.
Not sure what the workaround is though, please post one if you've worked something out.

**Comment by ThePademelon** [comment]
I'm not on ubuntu, but this message (`specified group 'uaccess' unknown`) appeared in my eudev system on startup so I commented out the same line as OP and fixed it.

**Comment by ronso0** [comment]
FWIW this is the PR that uncommented this line #813
and this added the "uaccess" tag #2558

The comment also says
> # Only some distribuions require the below

Is there a way to figure which distros require it?
